src/config.js

```diff
@@ -123,6 +118,29 @@ const GEMINI_INTEGRATION = {
   interactionChannel: process.env.AI_AUTOMATIC_INTERACTION_CHANNEL,
 };
 
+const CRON_REVIEW_SETTINGS = {
```

Comment:
maybe CRON_STATUS_REMINDER is a more specific name for this config, what do you think?